pkg/webui/components/safe-inspector/index.js

```diff
@@ -134,7 +137,16 @@ export class SafeInspector extends Component {
 
   @bind
   handleSwapToggle() {
-    this.setState(prev => ({ msb: !prev.msb }))
+    const { representation } = this.state
+    const { enableUint32 } = this.props
+
+    if (representation === 'msb') {
+      this.setState({ representation: 'lsb' })
```

Comment:
Consider something like this instead of nested ifs

```js
// Outside of the component.
const MSB = 'msb'
const LSB = 'lsb'
const UINT32_T = 'uint32_t'
const representationRotateMap = {
  [MSB]: LSB,
  [LSB]: UINT32_T,
  [UINT32_T]: MSB,
}

// Inside the component
_getNextRepresentation(current) {
  const { enableUint32 } = this.props
	  
  const next = representationRotateMap[current]
  return next === UINT32_T && !enableUint32 ? representationRotateMap[next] : next
}

@bind
handleSwapToggle() {
  this.setState(({ representation }) => ({
    representation: this._getNextRepresentation(representation),
  }))
}
```